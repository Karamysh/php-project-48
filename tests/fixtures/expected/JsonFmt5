{
  "status": "root",
  "children": {
    "common": {
      "status": "nested",
      "arrayValue": {
        "follow": {
          "status": "added",
          "addedValue": false
        },
        "setting1": {
          "status": "equal",
          "identialValue": "Value 1"
        },
        "setting2": {
          "status": "removed",
          "removedValue": 200
        },
        "setting3": {
          "status": "updated",
          "removedValue": true,
          "addedValue": null
        },
        "setting4": {
          "status": "added",
          "addedValue": "blah blah"
        },
        "setting5": {
          "status": "added",
          "addedValue": {
            "key5": "value5"
          }
        },
        "setting6": {
          "status": "nested",
          "arrayValue": {
            "doge": {
              "status": "nested",
              "arrayValue": {
                "wow": {
                  "status": "updated",
                  "removedValue": "",
                  "addedValue": "so much"
                }
              }
            },
            "key": {
              "status": "equal",
              "identialValue": "value"
            },
            "ops": {
              "status": "added",
              "addedValue": "vops"
            }
          }
        }
      }
    },
    "group1": {
      "status": "nested",
      "arrayValue": {
        "baz": {
          "status": "updated",
          "removedValue": "bas",
          "addedValue": "bars"
        },
        "foo": {
          "status": "equal",
          "identialValue": "bar"
        },
        "nest": {
          "status": "updated",
          "removedValue": {
            "key": "value"
          },
          "addedValue": "str"
        }
      }
    },
    "group2": {
      "status": "removed",
      "removedValue": {
        "abc": 12345,
        "deep": {
          "id": 45
        }
      }
    },
    "group3": {
      "status": "added",
      "addedValue": {
        "deep": {
          "id": {
            "number": 45
          }
        },
        "fee": 100500
      }
    }
  }
}
