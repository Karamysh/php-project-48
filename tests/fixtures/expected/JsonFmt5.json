{
  "status": "root",
  "children": [
    {
      "property": "common",
      "depth": 1,
      "status": "nested",
      "arrayValue": {
        "4": {
          "property": "follow",
          "depth": 2,
          "status": "added",
          "addedValue": false
        },
        "0": {
          "property": "setting1",
          "depth": 2,
          "status": "equal",
          "identialValue": "Value 1"
        },
        "1": {
          "property": "setting2",
          "depth": 2,
          "status": "removed",
          "removedValue": 200
        },
        "2": {
          "property": "setting3",
          "depth": 2,
          "status": "updated",
          "removedValue": true,
          "addedValue": null
        },
        "5": {
          "property": "setting4",
          "depth": 2,
          "status": "added",
          "addedValue": "blah blah"
        },
        "6": {
          "property": "setting5",
          "depth": 2,
          "status": "added",
          "addedValue": { "key5": "value5" }
        },
        "3": {
          "property": "setting6",
          "depth": 2,
          "status": "nested",
          "arrayValue": {
            "1": {
              "property": "doge",
              "depth": 3,
              "status": "nested",
              "arrayValue": [
                {
                  "property": "wow",
                  "depth": 4,
                  "status": "updated",
                  "removedValue": "",
                  "addedValue": "so much"
                }
              ]
            },
            "0": {
              "property": "key",
              "depth": 3,
              "status": "equal",
              "identialValue": "value"
            },
            "2": {
              "property": "ops",
              "depth": 3,
              "status": "added",
              "addedValue": "vops"
            }
          }
        }
      }
    },
    {
      "property": "group1",
      "depth": 1,
      "status": "nested",
      "arrayValue": [
        {
          "property": "baz",
          "depth": 2,
          "status": "updated",
          "removedValue": "bas",
          "addedValue": "bars"
        },
        {
          "property": "foo",
          "depth": 2,
          "status": "equal",
          "identialValue": "bar"
        },
        {
          "property": "nest",
          "depth": 2,
          "status": "updated",
          "removedValue": { "key": "value" },
          "addedValue": "str"
        }
      ]
    },
    {
      "property": "group2",
      "depth": 1,
      "status": "removed",
      "removedValue": { "abc": 12345, "deep": { "id": 45 } }
    },
    {
      "property": "group3",
      "depth": 1,
      "status": "added",
      "addedValue": { "deep": { "id": { "number": 45 } }, "fee": 100500 }
    }
  ]
}
